imports/ui/containers/ListPageContainer.jsx

```diff
@@ -3,7 +3,7 @@ import { withTracker } from 'meteor/react-meteor-data';
 import { Lists } from '../../api/lists/lists.js';
 import ListPage from '../pages/ListPage.jsx';
 
-const ListPageContainer = withTracker(({ match, menuOpen }) => {
+const ListPageContainer = withTracker(({ match }) => {
```

Comment:
We should probably wait for https://github.com/meteor/react-packages/pull/273 so we can avoid using `withTracker`

Reply:
Yes, that would be great to use useTracker. It's a pity that it takes so long to me merged...